tap_dynamics/streams.py

```diff
@@ -112,15 +112,16 @@ class IncrementalStream(BaseStream):
     replication_method = 'INCREMENTAL'
     batched = False
 
-    def get_records(self, max_pagesize: int = 100, bookmark_datetime: str = None):
+    def get_records(self, max_pagesize: int = 100, desired_columns: list = [], bookmark_datetime: str = None):
```

Comment:
its better to initialize list inside function. Kevin recently posted this in dev. https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments